x-pack/plugins/screenshotting/server/browsers/chromium/paths.ts

```diff
@@ -32,17 +34,35 @@ interface CommonPackageInfo extends PackageInfo {
   archivePath: string;
 }
 
+function isCommonPackage(p: PackageInfo): p is CommonPackageInfo {
+  return p.location === 'common';
+}
+
 export class ChromiumArchivePaths {
   public readonly packages: Array<CustomPackageInfo | CommonPackageInfo> = [
     {
       platform: 'darwin',
       architecture: 'x64',
-      archiveFilename: 'chromium-d163fd7-darwin_x64.zip',
-      archiveChecksum: '19aa88bd59e2575816425bf72786c53f',
-      binaryChecksum: 'dfcd6e007214175997663c50c8d871ea',
-      binaryRelativePath: 'headless_shell-darwin_x64/headless_shell',
-      location: 'custom',
-      revision: 856583,
+      archiveFilename: 'chrome-mac.zip',
```

Comment:
Have we given thought to the workflow of possible http errors with this at runtime?

Can we change the download URL, use a proxy, provide manual installation steps, disable entirely, etc?

Reply:
@jbudz with the exception of providing manual installation steps, those abilities would require additional kibana.yml settings. I'm not sure there are any other good solutions.

This PR only affects Mac users that generate PDF / PNG reports. Based on telemetry data, we know that Mac OS generates lower than 1% of all PDF and PNG reports.